library/init/coe.lean

```diff
@@ -74,7 +78,7 @@ has_coe_to_fun.coe
 @[reducible] def coe_sort {a : Sort u} [has_coe_to_sort.{u v} a] : a → has_coe_to_sort.S.{u v} a :=
 has_coe_to_sort.coe
 
-/- Notation -/
+/-! ## Notation -/
```

Comment:
`notation` declarations don't show up in the online docs, so it might be good to add a Notation section to the module doc too.
```suggestion
/-! ### Notation -/
```